jwst/skymatch/region.py

```diff
@@ -413,18 +413,25 @@ def intersection(self, edge):
         u = self._stop - self._start
         v = edge._stop - edge._start
         w = self._start - edge._start
-        D = np.cross(u, v)
 
-        if np.allclose(np.cross(u, v), 0, rtol=0,
+        # Find the determinant of the matrix formed by the vectors u and v
+        #    Note: Originally this was computed using a numpy "2D" cross product,
+        #          however, this functionality has been deprecated and slated for
+        #          removal.
+        D = np.linalg.det([u, v])
```

Comment:
This is numerically faster, but has a small numerical difference from using the `np.cross`. In my observations, the error is around machine precision.

To numerically match exactly,
```python
D = np.linalg.det(np.array([u, v]).transpose())
```
can be used, but this seems like overkill.